views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -36,14 +36,68 @@ define([
     'use strict';
 
     var ns = '.mathEntryInteraction';
+    var rootClass = 'mq-root-block';
+    var newLineClass = 'mq-tao-br';
+    var autoWrapClass = 'mq-tao-wrap';
+    var rootSelector = '.' + rootClass;
+    var newLineSelector = '.' + newLineClass;
+    var autoWrapSelector = '.' + autoWrapClass;
```

Comment:
Not sure whether that's just me, a best practice, or even a bad idea, but I prefer wrapping all these var in an objects like `cssClasses` and `selectors` to avoid creating to much variables at the top level scope and improve readability. What's your take on it?